ruby_on_rails/rspec.md

```diff
@@ -78,8 +78,9 @@ You should know exactly why you are adding each one of them and why is necessary
       driven_by :rack_test
     end
 
+    SELENIUM_DRIVER = ENV['DEBUG'].present? && ENV['DEBUG'].eql?('true') ? :selenium_chrome : :selenium_chrome_headless
     config.before(:each, type: :system, js: true) do
-      driven_by :selenium_chrome_headless
+      driven_by SELENIUM_DRIVER
```

Comment:
I specify the driver directly in the environment variable. default to `:selenium_chrome_headless`

```
config.before(:each, type: :system, js: true) do
  driven_by ENV['SELENIUM_DRIVER'].to_sym
end
```

I find the ENV variable `DEBUG` too generic. With a different name I'd be ok, but I'd prefer this solution because it allows you to also run it on firefox for example if you need to. See also https://github.com/renuo/so_many_devices

Reply:
@coorasse the advantage about `DEBUG` is, that selenium also knows this variable, so also its' output is verbose

Reply:
it seems like another good reason to keep them separate, no?

Reply:
I never used the verbose output of selenium. How does it look like?

Reply:
it looks very verbose ;-) i kind of changed my opinion. It's rather unhandy since it outputs every single request. @schmijos then i would go for @coorasse 's solution

Reply:
It doesn't matter so much for me. I normally use `driven_by :selenium_chrome_headless` directly in a before-block because normally I don't want to run **all** the tests in the browser but only selected ones.

So yes, go on.

Reply:
@coorasse if you are fine with this solution, i would merge